overlord/snapstate/handlers.go

```diff
@@ -528,21 +548,16 @@ func maybeRestart(t *state.Task, info *snap.Info) {
 	st := t.State()
 	if release.OnClassic && info.Type == snap.TypeOS {
 		t.Logf("Requested daemon restart.")
```

Comment:
These log messages seem misplaced for what's going on here. Should probably say "Requesting", or be moved down. Unrelated to this PR though.